src/nvim/tui/input.h

```diff
@@ -22,7 +23,6 @@ typedef struct term_input {
   RBuffer *key_buffer;
   uv_mutex_t key_buffer_mutex;
   uv_cond_t key_buffer_cond;
-  bool waiting_for_bg_response;
 } TermInput;
```

Comment:
Needs to come before `uv_mutex_t key_buffer_mutex;` - otherwise it will be false always, even if changed on 32bit.

Reply:
`uv_mutex_t` is `pthread_mutex_t` (via `pthread.h`).
Maybe the headers there are not correct for 32bit then?

Reply:
how about putting this next to `bool waiting` (line 14)

Reply:
Sure.  It was testing that https://github.com/neovim/neovim/pull/11073 actually fixes it.